modules/web-console/frontend/app/components/page-queries/components/queries-notebook/template.tpl.pug

```diff
@@ -62,6 +62,7 @@ mixin chart-settings
                 div
                     label Duration: #[b {{paragraph.duration | duration}}]
                     label.margin-left-dflt(ng-show='paragraph.localQueryMode') NodeID8: #[b {{paragraph.resNodeId | id8}}]
+                    i.btn.btn-default.fa.fa-trash-o(ng-click='clearResult(paragraph)' bs-tooltip='' data-title='Clear query result' data-trigger='hover' data-placement='bottom')
```

Comment:
Invalid icon for clear

Reply:
@nva  the icon choice looks reasonable, do you mean that we should strive not to use Font Awesome icons in WC?

Reply:
'trash' icon in other tools means delete records from database. For example: https://www.pgadmin.org/docs/pgadmin4/development/query_tool_toolbar.html#query-tool-toolbar
I would prefer 'erase' icon for this action

Reply:
@nva that makes sense.